cg/store/models.py

```diff
@@ -1077,5 +1077,7 @@ class IlluminaSampleRunMetrics(SampleRunMetrics):
     total_reads_in_lane: Mapped[BigInt | None]
     base_passing_q30_percent: Mapped[Num_6_2 | None]
     base_mean_quality_score: Mapped[Num_6_2 | None]
+    _yield: Mapped[BigInt | None] = mapped_column("yield", quote=True)
```

Comment:
why the underscore?

Reply:
yield is a reserved name and cannot be used like that

Reply:
Good solution 👍